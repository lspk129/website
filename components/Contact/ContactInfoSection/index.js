import styled from 'styled-components';

const ContactInfoSection = styled.div`
  display: grid;
  grid-column: 1 / 2;
  justify-content: center;
`;

export default ContactInfoSection;
