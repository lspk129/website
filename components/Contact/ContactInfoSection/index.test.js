import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import ContactInfoSection from './';

describe('ContactInfoSection', () => {
  const shallowWrapper = shallow(<ContactInfoSection />);

  it('renders correctly', () => {
    const wrapper = renderer.create(<ContactInfoSection />);
    const tree = wrapper.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders div element', () => {
    expect(shallowWrapper).toBePresent();
    expect(shallowWrapper).toHaveTagName('div');
  });
});
