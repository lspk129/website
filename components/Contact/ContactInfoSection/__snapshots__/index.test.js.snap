// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactInfoSection renders correctly 1`] = `
<div
  className="sc-bdVaJa cyDMwS"
/>
`;
